pkg/controllers/subnetset/subnetset_webhook.go

```diff
@@ -35,18 +35,22 @@ type SubnetSetValidator struct {
 
 func defaultSubnetSetLabelChanged(oldSubnetSet, subnetSet *v1alpha1.SubnetSet) bool {
```

Comment:
I'm thinking about maybe we should also not allow user to modify old labels on SubnetSet, otherwise our subnetset controller will update those old labels to new labels...

Reply:
good catch! I thought keeping the old label check, but not found the scenario